docs/user-guides/sbs_swarm_interface.md

```diff
@@ -0,0 +1,481 @@
+
+Add the helper functions `activate_led_bit_mask`,`deactivate_led_bit_mask` and the function`light_check` above `__main__`:
+```python
+def activate_led_bit_mask(scf):
+ scf.cf.param.set_value('led.bitmask', 255)
+
+def deactivate_led_bit_mask(scf):
+    scf.cf.param.set_value('led.bitmask', 0)
+
+def light_check(scf):
+    activateBitMask(scf)
+    time.sleep(2)
+    deactivateBitMask(scf)
+```
+`light_check` will light up a copter red for 2 seconds and then return them to normal.
+
+Below `... Swarm(...)`  in `__main__`, execute the light check for each copter:
+
+```python
+    swarm.parallel_safe(light_check)
+```
+The `light_check()` is going to be called through the `parallel_safe()` method which will execute it for for all Crazyflies in the swarm, in parallel. One thread per Crazyflie is started to execute the function. The threads are joined at the end and if one or more of the threads raised an exception this function will also raise an exception.
+
+```python
+import time
+
+import cflib.crtp
+from cflib.crazyflie.swarm import CachedCfFactory
+from cflib.crazyflie.swarm import Swarm
+
+
+def activate_led_bit_mask(scf):
+    scf.cf.param.set_value('led.bitmask', 255)
+
+def deactivate_led_bit_mask(scf):
+    scf.cf.param.set_value('led.bitmask', 0)
+
+def light_check(scf):
+    activate_led_bit_mask(scf)
+    time.sleep(2)
+    deactivate_led_bit_mask(scf)
+    time.sleep(2)
+
+
+uris = {
+    'radio://0/20/2M/E7E7E7E701',
+    'radio://0/20/2M/E7E7E7E702',
+    'radio://0/20/2M/E7E7E7E703',
+    'radio://0/20/2M/E7E7E7E704',
+    # Add more URIs if you want more copters in the swarm
+}
+
+if __name__ == '__main__':
+    cflib.crtp.init_drivers()
+    factory = CachedCfFactory(rw_cache='./cache')
+    with Swarm(uris, factory=factory) as swarm:
+        swarm.parallel_safe(light_check)
+
+```
+If everything is working properly, you can move to the next step .
+
+# Step 2: Security Before Flying
+Before executing any take off and flight maneuvers, the copters need to make sure that they have a precise enough position estimation. Otherwise it will take off anyway and it is very likely to crash. This is done through `reset_estimators()` by resetting the internal position estimator of each copter and waiting until the variance of the position estimation drops below a certain threshold. 
+```python
+with Swarm(uris, factory=factory) as swarm:
+        swarm.parallel_safe(lightCheck)
+        swarm.reset_estimators()
+```
+
+# Step 3: Taking off and Landing Sequentially
+Now we are going to execute the fist take off and landing using the high level commander. The high level commander is a class that handles all the high level commands like takeoff, landing, hover, go to position and others. You first need to enable it through the `commander.enHighLevel` parameter and then execute the take off and land commands through the below functions:
```

Comment:
perhaps an extra sentence why the high level commander is preferred (less communication,more autonomy on the crazyflie).  You can perhaps also refer somewhere to this doc that explans it: https://www.bitcraze.io/documentation/repository/crazyflie-firmware/master/functional-areas/sensor-to-control/commanders_setpoints/#high-level-commander but that can be also done on top.

Reply:
Yes, you are right, thank you!